desktop_version/src/Vlogging.c

```diff
@@ -5,6 +5,9 @@
 #ifdef _WIN32
 #   define WIN32_LEAN_AND_MEAN
 #   include <windows.h>
+#elif defined(__ANDROID__)
```

Comment:
In retrospect, I wonder if we could make the SDL_Log path a non-Android-specific path (`VLOG_USE_SDL`?) and for now Android will be the only user of that backend. Windows in particular might benefit from SDL's ability to route to both stdout as well as OutputDebugString.